cmd/test-utils/common.go

```diff
@@ -76,6 +78,25 @@ func (s *IntegrationSuite) RunStop(ctx context.Context) (*bytes.Buffer, error) {
 	return s.RunCommand(ctx, "stop")
 }
 
+func (s *IntegrationSuite) AllStopped() {
```

Comment:
I don't see the difference with [`StopTestSuite::allStopped()`](https://github.com/src-d/engine/pull/350/files#diff-6fd0a789630fde34454c1121b58bdf57R91)

Reply:
Answered in https://github.com/src-d/engine/pull/350#discussion_r266928822.

Reply:
I rebased the branch, now it's using the common code from #350.